src/hooks/useGoalSelection.ts

```diff
@@ -14,6 +14,7 @@ export const useGoalSelection = (goals: GoalItem[]): GoalItem | undefined => {
   const downPress = useKeyPress("ArrowDown");
   const rightPress = useKeyPress("ArrowRight");
   const leftPress = useKeyPress("ArrowLeft");
+  const spacePress = useKeyPress(" ");
```

Comment:
can we use "Space" key code to detect the space key press?

Reply:
@vinaybadgujar102 fixed it.